README.md

```diff
@@ -141,6 +141,10 @@ add `#define MLPACK_ENABLE_ANN_SERIALIZATION` before including `<mlpack.hpp>`.
 If you don't define `MLPACK_ENABLE_ANN_SERIALIZATION` and your code serializes a
 neural network, a compilation error will occur.
 
+***Warning:*** older versions of OpenBLAS (0.3.26 and older) may use too many
```

Comment:
Maybe add "... in the pthread build variant (where the openmp variant is fine) ..."  ?

Reply:
Done, thanks for suggesting this.